### PR TITLE
gabriel.danaila - pull request for project's documentation

### PR DESCRIPTION
**Pull request by Gabi10017**
### Pull Request Overview

This pull request adds the Project Documentation for DIY Pico Modulated Audio to Tesla Coil.

### TODO or Help Wanted

This pull request contains my project's documentation. 
Further modifications of the documentation will be uploaded here.

### Author

Signed-off-by: Gabriel Dănăilă <gdanaila10017@gmail.com>

### CODE REVIEW

File: docs/project/2024/gabriel.danaila/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+## Motivation
+
+I chose this project because I think it is an interesting, innovative idea to put in practice, that is neither too easy or too complex to do for a single-person project. This project idea was brought to me by the lab teacher.
+
+## Architecture 
+
+Add here the schematics with the architecture of your project. Make sure to include:
+ - what are the main components (architecture components, not hardware components)
+ - how they connect with each other
+
```
Comment (irina-b-dev) [inline docs/project/2024/gabriel.danaila/index.md:24]: Please add a block diagram of your project.

File: docs/project/2024/gabriel.danaila/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+
```
Comment (irina-b-dev) [inline docs/project/2024/gabriel.danaila/index.md:58]: Add all the components that you use

File: docs/project/2024/gabriel.danaila/index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
+
```
Comment (irina-b-dev) [inline docs/project/2024/gabriel.danaila/index.md:66]: Remove these libraries if you dont need them.

File: docs/project/2024/gabriel.danaila/index.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://example.com)
```
Comment (irina-b-dev) [inline docs/project/2024/gabriel.danaila/index.md:71]: Add some links for references

**Review by irina-b-dev — CHANGES_REQUESTED** [review]

File: docs/project/2024/gabriel.danaila/index.md (hunk 2 above)
Comment (Gabi10017) [reply docs/project/2024/gabriel.danaila/index.md:69]: Added.

File: docs/project/2024/gabriel.danaila/index.md (hunk 1 above)
Comment (Gabi10017) [reply docs/project/2024/gabriel.danaila/index.md:30]: Added.

File: docs/project/2024/gabriel.danaila/index.md (hunk 3 above)
Comment (Gabi10017) [reply docs/project/2024/gabriel.danaila/index.md:76]: Removed.

File: docs/project/2024/gabriel.danaila/index.md (hunk 4 above)
Comment (Gabi10017) [reply docs/project/2024/gabriel.danaila/index.md]: Added.

**Review by irina-b-dev — APPROVED** [review]

**Review by irina-b-dev — DISMISSED** [review]

**Review by DanutAldea — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Pull request merged**